src/emitter.ts

```diff
@@ -340,6 +341,15 @@ export function emitWebIDl(webidl: Browser.WebIdl, flavor: Flavor) {
         return obj.nullable ? makeNullable(resolvedType) : resolvedType;
     }
 
+    function nameWithForwardedTypes (i: Browser.Interface) {
+        const typeParameters = i["type-parameters"];
+
+        if (!typeParameters) return i.name;
+        if (!typeParameters.length) return i.name;
+
+        return `${i.name}<${typeParameters.map(t => t.replace(/\s+=.*$/, ''))}>`;
```

Comment:
i would rather we have two separate properties, one for the name of the type parameter, and one for the default.

Reply:
Sounds good 👍

Reply:
@mhegazy what do you think about this?

```typescript
"type-parameters"?: { name: string, default?: string }[]
```

Reply:
sounds good.